lms/static/sass/features/_course-experience.scss

```diff
@@ -145,6 +149,55 @@
   @include margin-left(0);
   @include padding-left($baseline);
 
+  // Course Goal Updates
+  .section-goals {
+    border: 1px solid $lms-border-color;
+    padding: $baseline*0.75;
+    border-radius: 5px;
+    position: relative;
+
+    &.hidden {
+      display: none;
+    }
+
+    h3 {
+      @include float(left); // For RTL purposes
+      @include margin-right($baseline/4);
+    }
+
+    select {
+      border: none;
+      background-color: transparent;
+      cursor: pointer;
```

Comment:
Why override the default behaviour for ```select```?

Reply:
In this case it was to make the select look like a read only field so that we didn't need to do the whole 'click and flip' with a readonly and a select field like on the learner profile.

Reply:
Could you not just set the ```select``` to ```disabled```?